docs/building-your-game.md

```diff
@@ -0,0 +1,74 @@
+# Building Your Game
+
+So, you have finished developing your game and you are ready to share your masterpiece with the world. This section will show you all that is needed to build and deploy your game.
+
+Here's what we will do:
+
+[[toc]]
+
+## Authoring
+
+Open the `Project` tab and select the `Authoring` side tab, here we can add the authoring details for your project.
+
+![](./images/buildingAndDeployment/building-your-game-authoring.png)
+
+1. **Name:** This will be the name of the executable or the .zip file when you export the game.
+2. **Developer:** Add the name of the developer here.
+3. **Homepage:** The URL to your website goes here.
+4. **Version:** Add a version number here to keep track of your game updates and patches.
+
+## Branding
+
+Open the `Project` tab, select the `Branding` side tab, here we have some branding options for the exported project.
+
+![](./images/buildingAndDeployment/building-your-game-branding.png)
+
+1. **Game icon:** Add an icon to your game, you can choose one from any of your imported textures.
+2. **Accent:** This changes the accent color of the loading screen.
+3. Here you can change whether the accent color affects the background or fill by inverting it. You can also hide the "Made with ct.js" loading screen here.
+
+## Render Options
+
+In the `Project` tab, select the `Render Options` side tab, in this area we have all available render specific options.
+
+![](./images/buildingAndDeployment/building-your-game-render-options.png)
+
+1. **Frame Rate:** This is the number of FPS (Frames Per Second) that your game will render at.
+2. Here you will find various options that affect how your game gets rendered.
+3. **Launch Mode:** When building a desktop version of your game, you can choose how the game gets launched.
+
+:::tip
+If you are making a pixel art game and you would like to preserve the sharp pixel look, you should make sure the `Disable image smoothing` option is checked.
+:::
+
+## Export Settings
+
+In the `Project` tab, select the `Export Settings`side tab.
+
+![](./images/buildingAndDeployment/building-your-game-export-settings.png)
+
+Here you can find some more advanced settings that will take effect when you export your game. For most projects the defaults will be fine.
+
+## Exporting your game
+
+In the `Main Menu` tab, you will see two export options under the `Deployment` heading.
+
+![](./images/buildingAndDeployment/building-your-game-deployment.png)
+
+### Export for web
+
+Exporting for the web is as easy as pressing the button. Once the game is finished exporting, you will be presented with the exported zip file containing your game.
+
+![](./images/buildingAndDeployment/building-your-game-builds-web.png)
+
+### Export for desktop
+
+Exporting for desktop is as easy as pressing the button and choosing the OS you want to build an executable for.
```

Comment:
Sounds repetitive (: better pick a different wording here.

Reply:
I have updated the wording for this.